### ISSUE TITLE
Compatibility

### ISSUE THREAD

**Issue by MCTyler** [CodeCrafter47/FreeBungeeChat#6]
I want to make this compatible with a quickshop plugin that listens to chat events to set up the shop. Can you help add this to the listener? https://github.com/KaiKikuchi/QuickShop/tree/master/src/main/java/org/maxgamer/quickshop/Listeners He added herochat but said he doesn't have time to add FreeBungeeChat. Any help would be appreciated! For now I have enabled the /g Global Chat Option but that is not ideal.

**Comment by MWFIAE** [comment]
The Problem with this is that FreeBungeeChat handles chatmessages on a bungeelevel, so a bukkit plugin will never be able to get the chat before it is send (and canceled) by FreeBungeeChat. The same problem comes with a quest-plugin I tested, citizens,  and many others. 

But yes this almost killed FreeBungeeChat as chat plugin for me, I ended up with modifying the code for my purpose, which broke many other things, but these weren't as important for me as the changes I made... 

An official solution is what we really need. But I'm not sure if this will ever come, because the last commit have been so long...
